src/index.ts

```diff
@@ -33,39 +35,38 @@ export interface CrawlResult {
 }
 
 interface CrawlOptions {
-  url: string;
+  url: url;
   crawl: boolean;
   results: LinkResult[];
   cache: Set<string>;
   checkOptions: CheckOptions;
 }
 
-/**
- * Instance class used to perform a crawl job.
- */
+const isHtml = (response: gaxios.GaxiosResponse) => {
+  const contentType = response.headers['content-type'] || '';
```

Comment:
So I don't mind losing the jsdoc specific annotations, but I *do* like the comments explaining to me what was happening here :)

Reply:
added back